src/commands/keyset.rs

```diff
@@ -38,25 +39,146 @@ pub struct Keyset {
     #[arg(short = 'c')]
     keyset_conf: PathBuf,
 
-    /// Domain name (only for create)
-    #[arg(short = 'n')]
-    domain_name: Option<Name<Vec<u8>>>,
-
-    /// State file (only for create)
-    #[arg(short = 's')]
-    keyset_state: Option<PathBuf>,
-
-    #[arg(short = 't')]
-    ttl: Option<u32>,
-
-    #[arg(short = 'b')]
-    bits: Option<usize>,
-
     /// Subcommand
-    #[arg()]
-    cmd: String,
+    #[command(subcommand)]
+    cmd: Commands,
+}
 
-    value: Option<String>,
+type OptDuration = Option<Duration>;
+
+#[derive(Clone, Debug, Subcommand)]
+enum Commands {
```

Comment:
Would it make sense to split this up into subcommands? For example make a `set` and `get` subcommand. I feel like that would make it easier to navigate.

Reply:
Yes. I'll take a look what that leads to.

Reply:
This needs careful design also with feedback from users. Keyset is mostly incompatible with the concepts in Yuri's paper, so we only try to explain that in the documentation. But it is indeed out of scope for this PR.